packages/frontend/src/components/common/Filters/FilterGroupForm.tsx

```diff
@@ -185,8 +185,8 @@ const FilterGroupForm: FC<Props> = ({
                         ]}
                         value={
                             isAndFilterGroup(filterGroup)
-                                ? FilterGroupOperator.and
-                                : FilterGroupOperator.or
+                                ? FilterGroupOperator.or
+                                : FilterGroupOperator.and
```

Comment:
Reading the condition, this fix make sense to me 🤔 
@almeidabbm I assume the bug is related to the new functions that "merge" groups. I will leave this PR for you to review

Reply:
can be waiting for him to review so that I can also understand where I went wrong